bundles/org.eclipse.equinox.p2.artifact.repository/src/org/eclipse/equinox/internal/p2/artifact/repository/simple/SimpleArtifactRepository.java

```diff
@@ -928,17 +937,23 @@ public IStatus getArtifacts(IArtifactRequest[] requests, IProgressMonitor monito
 				} catch (InterruptedException e) {
 					//ignore
 				}
+				overallStatus.addAll(overallStatus);
```

Comment:
If I understand this correctly, then the monitor of the DownloadJob (the `jobMonitor`) now only reports back the task details, but not consume any ticks.

So for example: The "main" job would stay at 10% done, spawn all the "helper" jobs and then jump to 100% once they are finished. Or am I overlooking any other connection between the "parent" and the "worker" progress monitors?